Client/Frontend/Settings/SettingsViewController.swift

```diff
@@ -581,6 +576,28 @@ class SettingsViewController: TableViewController {
         )
     }()
     
+    private func setUpSections() {
+        if #available(iOS 14.0, *), let settingsStore = walletSettingsStore {
+            settingsStore.isDefaultKeyringCreated { [weak self] created in
+                guard let self = self else { return }
+                var copyOfSections = self.sections
+                if created, let index = self.sections.firstIndex(where: {
+                    $0.uuid == self.featureSectionUUID.uuidString
+                }) {
+                    copyOfSections[index].rows.append(
+                        Row(text: Strings.Wallet.braveWallet, selection: { [unowned self] in
+                            let vc = UIHostingController(rootView: WalletSettingsView(settingsStore: settingsStore))
+                            self.navigationController?.pushViewController(vc, animated: true)
+                        }, image: #imageLiteral(resourceName: "menu-crypto").template, accessory: .disclosureIndicator)
+                    )
+                    self.dataSource.sections = copyOfSections
+                } else {
+                    self.dataSource.sections = self.sections
```

Comment:
Redundant else clause, just move the `self.dataSource.sections = copyOfSections` outside of the scope of this branch